shared/media-accelerator/get-started/get-started-sdk/project-implementation/linux-cpp.mdx

```diff
@@ -0,0 +1,29 @@
+<PlatformWrapper platform="linux-cpp">
+## Implement the <Vpd k="PRODUCT" /> sample project
+
+Take the following steps to download and compile the <Vpd k="NAME"/> sample project:
```

Comment:
This is technically correct, but I have no idea about the structure of the project or the calls that make this work. That info seems to be in the ref section where I need to analyse and guess what happens. Can you add something that explains how the code works please.

Reply:
Added C code